shared/utils/yaml-to-shorthand.py

```diff
@@ -425,11 +426,18 @@ def to_xml_element(self):
             if main_ref.attrib:
                 rule.append(main_ref)
 
-        # TODO: This is pretty much a hack, oval ID will be the same as rule ID
-        #       and we don't want the developers to have to keep them in sync.
-        #       Therefore let's just add an OVAL ref of that ID.
-        oval_ref = ET.SubElement(rule, "oval")
-        oval_ref.set("id", self.id_)
+        if self.external_oval:
+            check = ET.SubElement(rule, 'check')
+            check.set("system", "http://oval.mitre.org/XMLSchema/oval-definitions-5")
+            external_content = ET.SubElement(check, "check-content-ref")
+            if self.external_oval:
```

Comment:
this condition is true here, it is the same condition as line 429

Reply:
hahahaha thanks @jan-cerny! fixed.